src/asciidoctor-extensions-api.js

```diff
@@ -137,3 +148,18 @@ BlockProcessor.$$proto.named = function (name) {
 BlockProcessor.$$proto.onContext = function (context) {
   return this.$on_context(context);
 };
+
+/**
+ * @namespace
+ * @module Extensions/IncludeProcessor
+ */
+var IncludeProcessor = Extensions.IncludeProcessor;
+
+/**
+ * @memberof Extensions/IncludeProcessor
+ */
+IncludeProcessor.$$proto.handles = function (block) {
+  // FIXME: Dirty monkey patch !
+  IncludeProcessor.$$proto['$handles?'] = block;
```

Comment:
😉 
For the record, I just learned about `$$alloc.prototype`, I don't know if this could work as a workaround but the fix in core is 👍